src/ttsim/interface_dag_elements/fail_if.py

```diff
@@ -680,7 +682,7 @@ def backend_has_changed(
     if backend == "numpy":
         return
 
-    issues = ""
+    issue_list = []
```

Comment:
Please revert the strategy -- we actually had it like this at some point but it really is just one more thing to keep in mind.